packages/camera/lib/camera.dart

```diff
@@ -169,28 +221,27 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Initializes the camera on the device.
   ///
   /// Throws a [CameraException] if the initialization fails.
-  Future<Null> initialize() async {
+  Future<Null> openCamera() async {
```

Comment:
I would still use openCamera, it clearly states the intention (accessing the hardware to start the camera). And we already have a 'init' function in the plugin.

Reply:
I don't think "openCamera" states more intention than "initialize". Is it really being opened? could we open anything else than a camera? I would prefer to break current users as little as possible.
I agree that it can be confusing with `init` vs. `initialize`, but a user of the plugin usually never has to see `init`.